test/testscript.go

```diff
@@ -141,6 +184,39 @@ func (e *ProjectTest) ExecuteMakeTarget(target string) *ProjectTest {
 	return e
 }
 
+func (e *ProjectTest) NPMInstallDeps() *ProjectTest {
+	setupFunc := func(env *testscript.Env) error {
+		// Configure both npm and npx to only create and write files to the
+		// current working directory instead of the default $HOME directory
+		env.Setenv("NPM_CONFIG_CACHE", env.WorkDir+"/.npm")
+		env.Setenv("NPM_CONFIG_PREFIX", env.WorkDir+"/.npm-global")
+		env.Setenv("XDG_CACHE_HOME", env.WorkDir+"/.npx")
+		env.Setenv("XDG_CONFIG_HOME", env.WorkDir+"/.npx")
+		env.Setenv("XDG_DATA_HOME", env.WorkDir+"/.npx")
+
+		cmd := exec.Command("npm", "install")
+		cmd.Dir = env.WorkDir
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+		e.t.Logf("Running command: '%s'", cmd.String())
+		if err := cmd.Run(); err != nil {
+			return fmt.Errorf("failed to install npm deps: %v", err)
+		}
+		return nil
+	}
+	e.setupFuncs = append(e.setupFuncs, setupFunc)
+	return e
+}
+
+func (e *ProjectTest) SetEnv(key string, value string) *ProjectTest {
```

Comment:
This setup function can probably be removed.  I think it was only used in examples_test.go:41.

Reply:
removed